python/src/NAIS_doc.i.in

```diff
@@ -0,0 +1,78 @@
+%feature("docstring") OT::NAIS
+"Non Adaptative Importance Sampling (NAIS) algorhitm.
+
+Available constructors:
+    LHS(*event=ot.Event()*)
```

Comment:
you can omit the "Available constructors" lines since you have only one constructor

Reply:
alos: algorhitm

Reply:
yes you have reason i'm waiting for the last version of the doc